docs/howto/writing-migrations.txt

```diff
@@ -102,25 +131,18 @@ the respective field according to your needs.
             migrations.AddField(
                 model_name='mymodel',
                 name='uuid',
-                field=models.UUIDField(max_length=32, unique=True, default=uuid.uuid4),
+                field=models.UUIDField(unique=True, default=uuid.uuid4),
```

Comment:
I think I added argument sorting a while back. Do you mind putting `default` before `unique`.